platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/item/AGWorkflowItem.java

```diff
@@ -51,6 +52,15 @@ public Collection<Set<AGWorkflowTransitionExecution>> loadTransitionExecutionSet
 		return loadTransitionExecutionMap().getSets();
 	}
 
+	/**
+	 * Executes the longest possible cascade of auto transitions for this
+	 * {@link AGWorkflowItem}.
+	 */
+	public void executeCascadingAutoTransitions() {
```

Comment:
- the name may erroneously imply for someone that only cascading auto-transitions will be executed
- suggested names
  - `executeAllAutoTransitions`
  - `executeAllPossibleAutoTransitions`
  - `executeAutoTransitionsDeep`
  - `executeAutoTransitionsRecursively`

Reply:
Oped for `executeAllAutoTransitions`.